manim/mobject/svg/tex_mobject.py

```diff
@@ -621,6 +621,21 @@ def fade_all_but(self, index_or_string, opacity=0.5):
 
 
 class Title(Tex):
+    """
+    Example
+    --------
+    .. manim:: TitleExample
+        :save_last_frame:
+
+        import manim
```

Comment:
I think there is consensus that code blocks should be correctly formatted according to black as well. Please do that for this example as well. (PEP8 violations I can see here are the missing newline between imports and class definition, as well as some missing spaces around the assignment operators.)

Reply:
Yeah, in general I agree with this.
Having https://github.com/ManimCommunity/manim/issues/1484 in mind, I think it would be great to run the action once on all examples in order to get the formatting right.  
For this one, I can change it.